.run/chain-backend-server.run.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
what does this file use for?

Reply:
pre-configured 'run configuration' in GoLand